test/libp2p_secure_framed_stream_SUITE.erl

```diff
@@ -0,0 +1,151 @@
+-module(libp2p_secure_framed_stream_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-export([
+    all/0,
+    init_per_testcase/2,
+    end_per_testcase/2
+]).
+
+-export([
+    key_exchange/1,
+    stream/1
+]).
+
+%%--------------------------------------------------------------------
+%% COMMON TEST CALLBACK FUNCTIONS
+%%--------------------------------------------------------------------
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%   Running tests for this suite
+%% @end
+%%--------------------------------------------------------------------
+all() ->
+    [key_exchange, stream].
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%   Special init config for test case
+%% @end
+%%--------------------------------------------------------------------
+init_per_testcase(_, _Config) ->
+    test_util:setup(),
+    lager:set_loglevel(lager_console_backend, info),
+    _Config.
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%   Special end config for test case
+%% @end
+%%--------------------------------------------------------------------
+end_per_testcase(_, _Config) ->
+    ok.
+
+%%--------------------------------------------------------------------
+%% TEST CASES
+%%--------------------------------------------------------------------
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+key_exchange(_Config) ->
```

Comment:
We really need negative tests here, like where the far side is pretending to be the peer but it can't sign the ephemeral key, etc.